pyomo/contrib/mindtpy/MindtPy.py

```diff
@@ -191,7 +191,7 @@ class MindtPySolver(object):
         description="Tolerance on variable bounds."
     ))
     CONFIG.declare("zero_tolerance", ConfigValue(
-        default=1E-10,
+        default=1E-8,
```

Comment:
@jsiirola does Pyomo have constants for tolerances and maximum values such that we can use them here? In this case, we just used values that made sense but maybe there are some global constants we can set by default.

Reply:
@bernalde, Unfortunately, no.  Someone else (@qtothec? @michaelbynum?) was asking a similar question.  I think it would be reasonable to set up a "global" repository of default tolerances / constants (and make them settable through the Pyomo user configuration), but nothing exists at the moment.

Reply:
@jsiirola That was me. Hardest part is likely consistent naming and scoping. "zero_tolerance" to one person might be "feasibility_tolerance" to another. Frankly, the registry would help with that issue.